otx/cli/tools/find.py

```diff
@@ -15,17 +15,41 @@
 # and limitations under the License.
 
 import argparse
+import os
+
+from prettytable import PrettyTable
 
 from otx.cli.registry import Registry
+from otx.cli.utils.importing import get_backbone_registry, get_required_args
+
+# pylint: disable=too-many-locals
+
+SUPPORTED_TASKS = (
+    "CLASSIFICATION",
+    "DETECTION",
+    "INSTANCE_SEGMENTATION",
+    "SEGMENTATION",
+    "ACTION_CLASSIFICATION",
```

Comment:
I didn't know otx find can support action classification

Reply:
`otx find` it works fine for all template in `otx`!